### DISCUSSION TITLE
《宝可梦 白金》汉化修正补丁

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/PokemonChineseTranslationRevise#20] [General]
# 《宝可梦 白金》汉化修正补丁

https://blog.www.localhost/PokemonChineseTranslationRevise/Pt.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Hung 评论于 2021-06-29 14:22:07 (UTC+8)*****

1.41补丁无法正常升级

![https://d47jbcq60tnr6.cloudfront.net/2021629/5358-j2k6qi.74sfd.png](https://d47jbcq60tnr6.cloudfront.net/2021629/5358-j2k6qi.74sfd.png)

*****Xzonn 回复于 2021-06-29 21:00:57 (UTC+8)*****

是 ndstool 的问题，目前已修正，请下载最新版本的补丁。

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****胖軒 评论于 2021-11-10 03:26:06 (UTC+8)*****

請問最新的跟隨補丁能夠適用嗎?  
https://reurl.cc/52E1Mn  
順便問一下繁體的漢化有機會嗎？

*****Xzonn 回复于 2022-06-02 22:40:12 (UTC+8)*****

跟随补丁无法使用。因为我个人是简体中文使用者，暂时不考虑繁体化，如有需求可以自行修改代码。

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Mao Lin 评论于 2022-04-29 15:06:05 (UTC+8)*****

帷幕百货  販賣機的文字間格有被遮擋  還有3F商店的取消  是BUG嗎  還是文本就這樣設計?

![https://d47jbcq60tnr6.cloudfront.net/2022429/31987-1bzkvn0.x2rol.jpg](https://d47jbcq60tnr6.cloudfront.net/2022429/31987-1bzkvn0.x2rol.jpg)

![https://d47jbcq60tnr6.cloudfront.net/2022429/21611-3qmj5f.i96ru.jpg](https://d47jbcq60tnr6.cloudfront.net/2022429/21611-3qmj5f.i96ru.jpg)

*****Xzonn 回复于 2022-06-02 22:40:34 (UTC+8)*****

感谢反馈，应该是占位符的问题，下个版本尝试修复。

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****人世`难居 评论于 2022-05-21 22:14:54 (UTC+8)*****

首先非常感谢xzoon大佬给我们带来了四代官译,也算是给宝可梦中文化做出了巨大的贡献,但是,在重新游玩过程中,我发现了几处文本错误,都在图片中指出了,如果大佬有时间,有精力的话,希望能够改善一下,做到精益求精,给四代画上一个完美的句号,当然如果确实没有余力,也是无奈之举, 再次感谢xzoon大佬的新官译汉化,因为你们的存在才使得中文宝可梦圈得以不断壮大!致敬!

![https://d47jbcq60tnr6.cloudfront.net/2022521/14154-1vwff25.3s1p.png](https://d47jbcq60tnr6.cloudfront.net/2022521/14154-1vwff25.3s1p.png)

*****Xzonn 回复于 2022-06-02 22:42:48 (UTC+8)*****

感谢反馈，部分内容已修改，语句通顺的问题等我有空再整理一下。

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****梵·风 评论于 2022-12-21 09:31:49 (UTC+8)*****

请问有免通讯进化的补丁吗大佬们

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****虚吾伊德 评论于 2023-02-24 10:43:13 (UTC+8)*****

出现这种情况是怎么回事？求问！

![https://d47jbcq60tnr6.cloudfront.net/2023224/25073-19phvuc.tlq9.png](https://d47jbcq60tnr6.cloudfront.net/2023224/25073-19phvuc.tlq9.png)

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****胖胖 评论于 2023-03-11 00:46:07 (UTC+8)*****

白金，208道路的树果左边的标识牌翻译好像有问题，  
游戏里为：“有用的公告牌！埋下树果的话，当土的颜色变黑的时候，就要浇足够的水哦。除了那颜色外的土地，就用喷壶喷水吧！”  
神百给的翻译：“有用的公告牌！埋有树果的土地如果黑黑的话，说明水分充足。如果是其他颜色，就用喷壶浇点水吧！”

**Comment by uplush** [comment]
使用摇尾巴降低防御力时，动画好像会播放2遍

**Reply by Xzonn** [reply]
请提供详细的触发方式、运行环境、游戏版本，否则无法排查问题。

**Reply by Franinc** [reply]
如视频，一次摇尾巴技能会重复两次。水系泡沫技能偶尔也会攻击两次。
2.0.0 汉化，已检查日文版 ROM 的 MD5 无误，steam版retroarch + [melonds-ds 1.1.1](https://github.com/JesseTG/melonds-ds/releases)
https://github.com/Xzonn/PokemonChineseTranslationRevise/assets/26399054/e701d598-afca-4980-8686-872d15f51259

**Reply by Xzonn** [reply]
@Franinc 是否有稳定的触发方式，换日文版 ROM/其他模拟器是否也会触发该问题？

**Reply by Franinc** [reply]
技能施放两次看来是模拟器问题，日版在melonds-ds也施放两次，换到desmume 版本git b518fec后摇尾巴正常

**Reply by Franinc** [reply]
melonds-ds 核心选项 CPU Emulation 关闭 JIT Recompiler 后（开关需重启游戏）摇尾巴正常只一次。
JIT Recompiler下面包含子选项且默认全开，说不定两次施放bug和某子选项有关，还没测试。
![Snipaste_2024-05-02_19-15-33](https://github.com/Xzonn/PokemonChineseTranslationRevise/assets/26399054/a6fc1fcf-f3d9-44b9-9405-dcb1e6eb8b08)

**Reply by Franinc** [reply]
可能问题在Literal Optimizations，简单测试 三项全关或只关此项都没遇到。只关Branch Optimizations和只关Fast Memory都在一段时间后出问题（这问题还不是立刻触发）

**Reply by Xzonn** [reply]
@Franinc 感谢测试，我再研究研究。

**Comment by Rainzard** [comment]
飞叶快刀、地震、空气利刃、放电有时会连续攻击两次，第二次只有击中闪烁效果没有招式动画，双打时有时会对第一个攻击到的敌方连续攻击2次，享有独立的命中判定，似乎是aoe招式共有的问题？；天气系统有时在回合结束时循环生效（当前是日照，但是回合结束时提示日照状态后会连续变为降雨沙暴和冰雹，并会执行扣血，随后恢复为日照，然后本回合结束）

**Reply by Rainzard** [reply]
白金的神奥图鉴解锁不了151之后的内容，是不是和钻珍搞混了？

**Reply by Xzonn** [reply]
请提供详细的触发方式、运行环境、游戏版本，否则无法排查问题。

**Reply by Rainzard** [reply]
图鉴问题在同样运行环境下，日文版没有该问题，直接打开中文存档，或者日文原版重新开始之后通信传入151后的pm都可以正常显示151之后的内容，汉化2.0.0版本，附上存档（实际上解锁全国图鉴后依然显示不了神奥图鉴的151之后）
[Pocket.Monsters.Platinum(Japan)(Chinese Trans 2024)v2.0.0 - 解锁全国图鉴前.zip](https://github.com/Xzonn/PokemonChineseTranslationRevise/files/15186897/Pocket.Monsters.Platinum.Japan.Chinese.Trans.2024.v2.0.0.-.zip)
风怒的问题从melonDS0.9.5换到desmume 0.9.13没有成功复现，可能是模拟器的问题

**Reply by Xzonn** [reply]
图鉴问题已确认，待定位问题；招式重复生效的问题似乎都是在 melonDS 模拟器上发生的，需要进一步收集信息。

**Reply by Xzonn** [reply]
图鉴问题已在 #25 中修复。
